packages/lib-utils/src/k8s/ws-factory.ts

```diff
@@ -0,0 +1,332 @@
+import { consoleLogger } from '@monorepo/common';
```

Comment:
Andrew is working on migrating this code. I just moved it over as a placeholder API.